src/plugins/intel_gpu/include/intel_gpu/graph/network.hpp

```diff
@@ -6,13 +6,13 @@
 
 #include "intel_gpu/graph/topology.hpp"
 #include "intel_gpu/graph/program.hpp"
+#include "intel_gpu/graph/serialization/binary_buffer.hpp"
```

Comment:
https://github.com/openvinotoolkit/openvino/pull/13801#discussion_r1017858297
Moved the include path. Thank you.